modern/src/maki-interpreter/interpreter.js

```diff
@@ -160,7 +160,14 @@ export function* interpret(start, program, stack = []) {
           methodArgs.push(aValue);
         }
         const obj = popStackValue();
-        const ret = obj[methodName](...methodArgs);
+        let ret;
+        if (obj === 0) {
```

Comment:
This is where we were erroring due to the [Js]WinampAbstractionLayer bug, right?